test/functional/site/server.js

```diff
@@ -48,11 +48,19 @@ Server.prototype._setupRoutes = function () {
     this.app.get('*', function (req, res) {
         var reqPath      = req.params[0] || '';
         var resourcePath = path.join(server.basePath, reqPath);
+        var delay        = typeof req.query.delay === 'number' && parseInt(req.query.delay, 10);
```

Comment:
it's not good when a variable can have different types. Now it's `boolean` or `number`.
It should be number or null|number.
If delay is string you can write
```js
    var delay = req.query.delay ? parseInt(req.query.delay) : 0;
```
If it's a number just
```js
var delay = req.query.delay : 0; // as it was in the v.15.1
```

And you can get rid of unnecessary condition `if (delay) ` below